app/services/alerts/error_attributes_factory.rb

```diff
@@ -0,0 +1,23 @@
+module Alerts
+  class ErrorAttributesFactory
```

Comment:
This could be a Builder rather than a Factory - or even could be added to the AlertError Class